cli/tools/test/mod.rs

```diff
@@ -399,6 +399,33 @@ impl TestFailure {
     }
   }
 
+  pub fn error_location(&self) -> Option<TestLocation> {
+    match self {
+      // TODO: What happens if not enough call stack frames were recorded?
+      // TODO: Could anything else trigger the ext:cli/40_test.js file? Maybe we should also test for the method name
+      TestFailure::JsError(js_error) => js_error
+        .frames
+        .iter()
+        .position(|v| v.file_name.as_deref() == Some("ext:cli/40_test.js"))
```

Comment:
Maybe we should weaken this to `v.file_name.as_deref().is_some_and(|s| s.starts_with("ext:"))`, so it's less brittle wrt internal changes

Reply:
Sounds good, am doing the change

Reply:
I did the necessary testing and it turns out that we should be checking for `ext:cli/40_test.js`. Turns out that there are a lot of edge cases.